stripe-eventbridge-lambda/README.md

```diff
@@ -0,0 +1,115 @@
+
+Amazon CloudWatch Log Groups are provisioned for debugging and auditing of all Stripe events as well as the failed payment events specifically. This pattern deploys two EventBridge rules, one Lambda function, two CloudWatch Log Groups, and a DynamoDB table to store the failed payment details.
+
+Learn more about this pattern at Serverless Land Patterns: https://serverlessland.com/patterns/stripe-eventbridge-lambda
+
+Important: this application uses various AWS services and there are costs associated with these services after the Free Tier usage - please see the [AWS Pricing page](https://aws.amazon.com/pricing/) for details. You are responsible for any AWS costs incurred. No warranty is implied in this example.
+
+## Requirements
+
+* [Create an AWS account](https://portal.aws.amazon.com/gp/aws/developer/registration/index.html) if you do not already have one and log in. The IAM user that you use must have sufficient permissions to make necessary AWS service calls and manage AWS resources.
+* [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2.html) installed and configured
+* [Git Installed](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
+* [AWS CDK CLI](https://docs.aws.amazon.com/cdk/v2/guide/getting_started.html) (AWS CDK) installed
+* [Create a Stripe account](https://stripe.com/register) if you do not already have one and log in.
+* [Set up the Stripe Amazon EventBridge SaaS integration](https://docs.stripe.com/event-destinations/eventbridge) if you have not already configured the integration.
+
+## Deployment Instructions
+
+1. Create a new directory, navigate to that directory in a terminal and clone the GitHub repository:
+    ``` 
+    git clone https://github.com/aws-samples/serverless-patterns
+    ```
+2. Change directory to the pattern directory:
+    ```
+    cd stripe-eventbridge-lambda
+    ```
+3. From the command line, use AWS CDK to deploy the AWS resources for the pattern as specified in the app.py file. A command-line argument is needed to deploy the CDK stack, "stripeEventBusName". This argument should be the name of the **SaaS event bus** associated with your Stripe [partner event source](https://docs.aws.amazon.com/eventbridge/latest/userguide/eb-saas.html).
+    ```
+    cdk deploy --parameters stripeEventBusName=SAAS_EVENT_BUS_NAME_HERE
```

Comment:
is this missing any instructions? I am getting the below error

/serverless-patterns/stripe-eventbridge-lambda/app.py", line 3, in <module>
    import aws_cdk as cdk
ModuleNotFoundError: No module named 'aws_cdk'

Reply:
I don't believe so, I do not see that error

Reply:
let me take a deeper look